### PR TITLE
Update LibreWolf.LibreWolf to v102.0.1-1

### PR DESCRIPTION
**Pull request by maltejur**
(This pull-request was auto-generated.)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65556)

### CODE REVIEW

File: manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - htm
+  - html
+  - url
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VC++2015-2019Redist-x64
+      MinimumVersion: 14.28.29913.0
+Installers:
+  - InstallerLocale: en-US
+    Architecture: x86
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml:20]:
```suggestion
    Architecture: x64
```

File: manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    InstallerType: nullsoft
+    Scope: machine
+    InstallerUrl: "https://gitlab.com/librewolf-community/browser/windows/uploads/d38454411b712bef93312c8dc2ef98b7/librewolf-102.0.1-1.en-US.win64-setup.exe"
+    InstallerSha256: "1cc4ee8be2dda030c728f1910dbb507a3eb611d0e1319cad9fb410a0144af6c9"
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml:24]:
```suggestion
    InstallerUrl: https://gitlab.com/librewolf-community/browser/windows/uploads/d38454411b712bef93312c8dc2ef98b7/librewolf-102.0.1-1.en-US.win64-setup.exe
    InstallerSha256: 1CC4EE8BE2DDA030C728F1910DBB507A3EB611D0E1319CAD9FB410A0144AF6C9
```

File: manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: LibreWolf.LibreWolf
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.installer.yaml:1]: Please update your automation to add the Schema Headers and also to use the 1.1.0 schema
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json

PackageIdentifier: LibreWolf.LibreWolf
```

File: manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: LibreWolf.LibreWolf
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.locale.en-US.yaml:1]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json

PackageIdentifier: LibreWolf.LibreWolf
```

File: manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: LibreWolf.LibreWolf
```
Comment (Trenly) [inline manifests/l/LibreWolf/LibreWolf/102.0.1-1/LibreWolf.LibreWolf.yaml:1]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json

PackageIdentifier: LibreWolf.LibreWolf
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by maltejur** [comment]
Thanks for the review.

> Please update your automation

Will also do that

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**